.github/build.sh

```diff
@@ -232,5 +232,4 @@ fi
 cmake --build . --target test-aws-iot-device-client
 
 ### Run Tests ###
-env AWS_CRT_MEMORY_TRACING=1 ./test/test-aws-iot-device-client
-./test/test-aws-iot-device-client --gtest_filter=LogQueueTest.notifyAllOnShutdown --gtest_repeat=1000
```

Comment:
@RogerZhongAWS The notify unit test has been repeating for 1k times over the past week and we haven't seen any flaky tests.  I think it is safe to remove the repeat. Please let me know if you disagree.

Reply:
Agreed